package.json

```diff
@@ -100,6 +100,7 @@
     "lorem-ipsum": "^1.0.6",
     "merge-stream": "^2.0.0",
     "node-sass": "^4.13.1",
+    "node-sass-package-importer": "^5.3.2",
```

Comment:
Should not need this, see build file notes.

Reply:
closing this thread in favor of conversation on other thread / same topic:
https://github.com/patternfly/patternfly-elements/pull/895#discussion_r444896342